src/DAL/migration/materializedViewPolygonParts.sql

```diff
@@ -0,0 +1,75 @@
+-- View: PolygonParts.polygon_parts
+-- DROP MATERIALIZED VIEW IF EXISTS "PolygonParts".polygon_parts;
+CREATE MATERIALIZED VIEW IF NOT EXISTS "PolygonParts".polygon_parts TABLESPACE pg_default AS WITH joined_parts AS (
+   SELECT row_number() OVER (
+         PARTITION BY polygon_parts.geom
+         ORDER BY parts."internalId" DESC
+      ) AS row_num,
+      polygon_parts.geom,
+      parts."internalId",
+      parts."recordId",
+      parts."productId",
+      parts."productName",
+      parts."productVersion",
+      parts."ingestionDate",
+      parts."sourceDateStart",
+      parts."sourceDateEnd",
+      parts."minResolutionDeg",
+      parts."maxResolutionDeg",
+      parts."minResolutionMeter",
+      parts."maxResolutionMeter",
+      parts."minHorizontalAccuracyCE90",
+      parts."maxHorizontalAccuracyCE90",
+      parts.sensors,
+      parts.region,
+      parts.classification,
+      parts.description,
+      parts."imageName",
+      parts."productType",
+      parts."srsName"
+   FROM (
+         SELECT (
+               st_dump(
```

Comment:
This means that we store `multipolygons` and only return the parts as seperate? Why not store them as different parts to begin with?